APIs/openEO/openEO.qmd

```diff
@@ -0,0 +1,37 @@
+---
+title: "openEO"
+format:
+  html:
+    code-fold: true
+jupyter: python3
+---
+
+## Overview
+
+openEO provides intuitive programming libraries that enable easy analysis of diverse earth observation datasets. These libraries facilitate efficient access and processing of large-scale data across multiple infrastructures. This broad compatibility supports a wide range of applications, including exploratory research and the generation of detailed maps and information derived from earth observation.
+
+Developers can utilize openEO APIs to build applications that seamlessly access and process data from different sources, eliminating the requirement to learn specific access methods for each source. This streamlined approach simplifies the development of innovative applications and services that leverage Earth Observation data.
+
+## Added-value of openEO API
+
+The key benefits of using openEO API can be summarized as follows:
+
+* Unified and straightforward access to multiple Earth observation datasets.
+* Scalable and efficient processing capabilities.
+* A standardized system that works across different platforms.
+* Independence from underlying technologies and software libraries.
+* Reproducibility through transparent workflows, supporting principles of FAIR (Findable, Accessible, Interoperable, and Reusable) and Open Science.
+
+When using the openEO API, users can choose [JavaScript](JavaScript_Client/JavaScript.qmd), [Python](Python_Client/Python.qmd), or [R](R_Client/R.qmd) as their client library. This allows them to work with any back-end and compare them based on capacity, cost, and result quality.
+
+Nevertheless, if you are not familiar with programming, you could start using the [web-based editor for openEO](#). It supports visual modelling of your algorithms and simplified JavaScript-based access to the openEO workflows and providers. An overview of the openEO web-editor is available in the [Application](https://documentation.dataspace.copernicus.eu/Applications.html) section of this documentation. 
+
+## Datacubes
+
+In openEO, a datacube is a fundamental concept and a key component of the platform. Data is represented as datacubes in openEO, which are multi-dimensional arrays with additional information about their dimensionality. Datacubes can provide a nice and tidy interface for spatiotemporal data as well as for the operations you may want to execute on them. An in-depth introduction to datacubes and processing them with openEO can be found [here](https://openeo.org/documentation/1.0/datacubes.html){target='_blank'}
+
+## Contribute
```

Comment:
I would remove this section, users can certainly contribute via openeo.org,but for CDSE they should rather report issues via the to be created CDSE forum.